include/superior_mysqlpp/converters/to_integer.hpp

```diff
@@ -121,9 +177,19 @@ namespace SuperiorMySqlpp { namespace Converters
             }
         };
 
+        /**
+         * @brief Converts string into unsigned 32 bits long integer.
+         * @tparam validate Check if string is convertible.
+         */
         template<bool validate>
         struct ToIntegerUnsignedImpl<std::uint32_t, validate>
         {
+            /**
+             * @brief Parses string to unsigned 32 bits long integer.
```

Comment:
*unsigned 32 bits long integer* - IMHO replacing this part of sentence by `uint32_t` should be descriptive enough, thoughts?

Reply:
for other types as well

Reply:
ack